src/core/Wyam.Configuration/ConfigScript/ScriptManager.cs

```diff
@@ -8,12 +8,26 @@
 using Microsoft.CodeAnalysis.CSharp;
 using Microsoft.CodeAnalysis.Emit;
 using Microsoft.CodeAnalysis.Text;
+
+using Wyam.Common.Execution;
 using Wyam.Common.Tracing;
 using Wyam.Core.Execution;
 
 namespace Wyam.Configuration.ConfigScript
 {
-    internal class ScriptManager
+    internal interface IScriptManager
```

Comment:
+1 for making an interface to make testing the `CacheManager` easier. Let's move it to a `IScriptManager.cs` file.

Reply:
I was wondering about that, I'll move it out.